test/elixir/test/all_docs_test.exs

```diff
@@ -41,11 +41,9 @@ defmodule AllDocsTest do
     assert resp["total_rows"] == length(rows)
 
     # Check _all_docs offset
-    retry_until(fn ->
```

Comment:
I assume the removal of retry_until is because fdb is more consistent than old clustering code? We'll want to dedent the inner block though right? mix format should take care of that I think?

Reply:
That is right